src/main/java/io/commercetools/sunrise/steps/Main.java

```diff
@@ -11,6 +11,9 @@ public static void main(String[] args) throws Throwable {
         final List<String> usedArgsList = new LinkedList<>();
         usedArgsList.add("--glue");
         usedArgsList.add("io.commercetools.sunrise.steps");
+        //Ignores all scenarios or features tagged with @ignore
+        usedArgsList.add("--tags");
+        usedArgsList.add("~@ignore");
```

Comment:
@hajoeichler ok thanks for the info! changed: bdbc6fb47b733151b36031e9f6e78304bb6bb107